packages/aws-cdk/lib/init-templates/app/python/setup.py

```diff
@@ -0,0 +1,50 @@
+import json
+import setuptools
+
+
+with open("README.md") as fp:
+    long_description = fp.read()
+
+
+setuptools.setup(
+    name="hello",
+    version="0.0.1",
+
+    description="A sample CDK Python app",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+
+    author="author",
+
+    package_dir={"": "hello"},
+    packages=setuptools.find_packages(where="hello"),
+
+    install_requires=[
```

Comment:
Does it make sense in python-land to have this duplicated with requirements.txt?

Reply:
Many packages use both but I have simplified this to put the requirements in the `setup.py` and then tell pip to use those rather than list them explicitly in `requirements.txt`.